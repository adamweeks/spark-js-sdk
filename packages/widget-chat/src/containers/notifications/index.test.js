import React from 'react';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';

import store from '../../store';

import Notifications from '.';

describe(`Notifications container`, () => {
  it(`renders properly`, () => {
    const component = renderer.create(
      <Provider store={store}>
        <Notifications />
      </Provider>
    );
    expect(component).toMatchSnapshot();
  });
});
