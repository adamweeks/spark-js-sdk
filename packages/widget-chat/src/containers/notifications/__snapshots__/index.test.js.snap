exports[`Notifications container renders properly 1`] = `null`;
